core/templates/dev/head/domain/question/QuestionBackendApiService.ts

```diff
@@ -25,14 +25,21 @@ angular.module('oppia').factory('QuestionBackendApiService', [
       $http, $q, UrlInterpolationService, QUESTIONS_LIST_URL_TEMPLATE,
       QUESTION_PLAYER_URL_TEMPLATE) {
     var _fetchQuestions = function(
-        skillIds, questionCount, successCallback, errorCallback) {
+        skillIds, questionCount, questionsSortedByMastery,
+        successCallback, errorCallback) {
       if (!validateRequestParameters(skillIds, questionCount, errorCallback)) {
         return;
       }
+      if (questionsSortedByMastery) {
```

Comment:
Do you need this conversion? This should just show as 'true'/'false' right?

Reply:
true and false in Javascript are booleans instead of strings. If not converted, it will raise an error saying the URL contains something other than strings.

Reply:
Done.